packaging/installer/README.md

```diff
@@ -356,7 +356,8 @@ pkg install e2fsprogs-libuuid
 pkg install libuv
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/Judy-1.0.5_2.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/python36-3.6.9.txz
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.15.0.txz
+ln -s /usr/local/lib/libjson-c.so /usr/local/lib/libjson-c.so.4
```

Comment:
This is not what I see when I test it on 2.4.4-RELEASE.
After installing the json-c package I have `/usr/local/lib/libjson-c.so -> libjson-c.so.3.0.1`, but I also have the problem that `service onestart netdata` does not start the system and I cannot find any errors. Still investigating...

Reply:
Resolved with @thiagoftsm over slack - instructions are correct and testing has been replicated.